package.json

```diff
@@ -70,7 +72,9 @@
     "mocha": "~3.5.3",
     "mock-require": "~2.0.2",
     "redux-devtools": "~3.4.0",
-    "sinon": "~4.0.2",
-    "style-loader": "~0.19.0"
+    "source-map-loader": "~0.2.3",
+    "style-loader": "~0.19.0",
+    "typescript": "~2.6.1",
```

Comment:
Not in alphabetical order - don't manually edit package.json by hand, use npm install to do so

Reply:
There was a merge conflict, sorry.